server/ai_live_video.go

```diff
@@ -33,4 +36,39 @@ func startTricklePublish(url *url.URL, params aiRequestParams) {
 }
 
 func startTrickleSubscribe(url *url.URL, params aiRequestParams) {
+	// subscribe to the outputs and send them into LPMS
+	subscriber := trickle.NewTrickleSubscriber(url.String())
+	r, w, err := os.Pipe()
+	if err != nil {
+		slog.Info("error getting pipe for trickle-ffmpeg", "url", url, "err", err)
+	}
+
+	// read segments from trickle subscrption
```

Comment:
```suggestion
	// read segments from trickle subscription
```

Reply:
fixed in https://github.com/livepeer/go-livepeer/pull/3232/commits/91282eec9d4b5eda97cfb16c1996b9a53ba10e73